src/atoms/LinkWrapper/LinkWrapper.js

```diff
@@ -0,0 +1,50 @@
+import React, { PropTypes } from 'react';
+import styles from './link_wrapper.module.scss';
+import classnames from 'classnames';
+import _ from 'lodash';
+
+function LinkWrapper ( props ) {
+  const {
+    className,
+    children,
+  } = props;
+
+  return (
+    <a
+      { ..._.omit(props, ['children', 'className']) }
```

Comment:
is the only prop you're using here `href`? if so you could declare that explicitly to help with readability